pkg/scheduler/internal/cache/cache.go

```diff
@@ -237,6 +241,9 @@ func (cache *cacheImpl) UpdateSnapshot(nodeSnapshot *Snapshot) error {
 			if (len(existing.PodsWithRequiredAntiAffinity) > 0) != (len(clone.PodsWithRequiredAntiAffinity) > 0) {
 				updateNodesHavePodsWithRequiredAntiAffinity = true
 			}
+			if !reflect.DeepEqual(existing.PVCRefCounts, clone.PVCRefCounts) {
```

Comment:
I think we should (1) check if updateUsedPVCSet is already false, and (2) just verify the entires are identical (the int value doesn't quite matter)


```go
			if !updateUsedPVCSet {
				if len(existing.PVCRefCounts) != len(clone.PVCRefCounts) {
					updateUsedPVCSet = true
				} else {
					for pvcKey := range clone.PVCRefCounts {
						if _, ok := existing.PVCRefCounts[pvcKey]; !ok {
							updateUsedPVCSet = true
							break
						}
					}
				}
			}
```

Reply:
Sounds good, fixed.